app/src/main/java/com/epfl/beatlink/ui/search/components/Cards.kt

```diff
@@ -0,0 +1,189 @@
+package com.epfl.beatlink.ui.search.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epfl.beatlink.R
+import com.epfl.beatlink.model.profile.ProfileData
+import com.epfl.beatlink.model.spotify.objects.SpotifyTrack
+import com.epfl.beatlink.ui.theme.PrimaryGradientBrush
+import com.epfl.beatlink.ui.theme.PrimaryPurple
+
+@Composable
+fun ProfileCard(profile: ProfileData) {
+  Card(modifier = Modifier.testTag("profileCardItem").width(65.dp)) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier =
+            Modifier.testTag("profileCardColumn").fillMaxSize().background(color = Color.White)) {
+          Image(
+              painter = painterResource(id = R.drawable.default_profile_picture),
+              contentDescription = null,
+              modifier = Modifier.testTag("profileCardImage").fillMaxWidth().height(65.dp))
+
+          Text(
+              text = profile.name ?: "",
+              style =
+                  TextStyle(
+                      fontSize = 10.sp,
+                      lineHeight = 20.sp,
+                      fontFamily = FontFamily(Font(R.font.roboto)),
+                      fontWeight = FontWeight(700),
+                      color = PrimaryPurple,
+                      textAlign = TextAlign.Center,
+                      letterSpacing = 0.1.sp,
+                  ),
+              modifier = Modifier.testTag("profileCardName").background(color = Color.White))
```

Comment:
here you should use the app backgroup (eg. lightThemeBackground) instead of plain white